toolchain/sem_ir/dump.cpp

```diff
@@ -247,6 +247,7 @@ LLVM_DUMP_METHOD auto Dump(const File& file,
 LLVM_DUMP_METHOD auto Dump(const File& file, SpecificId specific_id) -> void {
   DumpNoNewline(file, specific_id);
   llvm::errs() << '\n';
+  Dump(file, file.specifics().Get(specific_id).args_id);
```

Comment:
Sorry, quick question -- comparing with #5140, do you want to add indent or adjust the newline above?

(no strong opinion here, just noting the difference in approach)

Reply:
I don't, because this is a multi-line output, so indenting here would only affect one line. We'd need to add indent parameters to a Dump method.